yew/src/virtual_dom/vtag.rs

```diff
@@ -1017,6 +1022,74 @@ mod tests {
         html! { <div><a data-val=Box::<u32>::default() /></div> };
     }
 
+    #[test]
+    fn it_does_not_set_empty_class_name() {
+        let parent = document().create_element("div").unwrap();
+
+        #[cfg(feature = "std_web")]
+        document().body().unwrap().append_child(&parent);
+        #[cfg(feature = "web_sys")]
+        document().body().unwrap().append_child(&parent).unwrap();
+
+        let mut elem = html! { <div class=""></div> };
+        elem.apply(&parent, None, None);
+        let vtag = assert_vtag(&mut elem);
+        // test if the className has not been set
+        assert!(!vtag.reference.as_ref().unwrap().has_attribute("class"));
+    }
+
+    #[test]
+    fn it_does_not_set_missing_class_name() {
+        let parent = document().create_element("div").unwrap();
+
+        #[cfg(feature = "std_web")]
+        document().body().unwrap().append_child(&parent);
+        #[cfg(feature = "web_sys")]
+        document().body().unwrap().append_child(&parent).unwrap();
+
+        let mut elem = html! { <div></div> };
```

Comment:
Can you also add a test for `class=None`

Reply:
I tried adding a test for `None`, but the compiler could not know which type the Option has.
I will add a test for `class=Option::<&str>::None`.
I will also add a test using a variable of type `Option<&str>`, but containing `None`.

Reply:
Ah, that works, thanks!